open-craft/cs_comments_service
# [BB-2386] Mongo authentication mechanism configurable from the environment

Issue: [BB-2386](https://tasks.opencraft.com/browse/bb-2386)

Cherry picked: [fafb9e](https://github.com/edx/cs_comments_service/pull/316/commits/fafb9e8152fd3ec0ea5326b2603f545e8acf585d)

Testing Instructions:
- Try creating a new instance and check for configuration settings of it

Reviewers:
